sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-jedis/pom.xml

```diff
@@ -16,8 +16,8 @@
   <modelVersion>4.0.0</modelVersion>
 
   <groupId>com.azure</groupId>
-  <artifactId>azure-messaging-eventhubs-checkpointstore-redis</artifactId>
-  <version>1.0.0-beta.1</version> <!-- {x-version-update;com.azure:azure-messaging-eventhubs-checkpointstore-redis;current} -->
+  <artifactId>azure-messaging-eventhubs-checkpointstore-jedis</artifactId>
```

Comment:
You need to follow the versioning guidelines and change this in the corresponding version_client.txt files.  https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#versions-and-versioning